systemvm/debian/opt/cloud/bin/configure.py

```diff
@@ -918,15 +918,6 @@ def forward_vpc(self, rule):
         if not rule["internal_ports"] == "any":
             fw_prerout_rule += ":" + self.portsToString(rule["internal_ports"], "-")
 
-        fw_postrout_rule = "-A POSTROUTING -d %s/32 " % rule["public_ip"]
```

Comment:
pylint error: no longer defined but still used in 942

Reply:
should be fixed now